dali/operators/random/uniform.cc

```diff
@@ -75,7 +75,7 @@ This argument is mutually exclusive with ``values``.
 This argument is mutually exclusive with ``range``.
 )code", std::vector<float>({}))
   .AddOptionalArg("shape",
-    R"code(Shape of the samples.)code", std::vector<int>{1});
+    R"code(Shape of the samples.)code", std::vector<int>{1}, true);
```

Comment:
```suggestion
    R"code(Shape of the samples.)code", std::vector<int>{}, true);
```
The default has been changed to generate _scalars_, not 1-element 1D tensors. It should stay like this!